src/modules/AutoinstLVM.rb

```diff
@@ -559,6 +560,22 @@ def Write
         end
       end
       Builtins.y2milestone("targetmap: %1", @targetMap)
+
+      AutoinstStorage.AutoTargetMap.each do |device, data|
+        target_map = Storage.GetTargetMap()
+        if target_map.has_key?(device) && data.fetch("type", :CT_DISK) == :CT_LVM
+          if data["enable_snapshots"] && target_map[device].has_key?("partitions")
+            root_partition = target_map[device]["partitions"].find {
```

Comment:
So just because I split a long line, see AutoyastStorage.rb, I also have to change the syntax. That's sick.

Not going to retest the code.